packages/domains-table/src/domains-table/style.scss

```diff
@@ -259,6 +259,9 @@
 }
 
 .domains-table-mobile-cards {
+	overflow-y: auto;
+	max-height: calc(100vh - 326px);
```

Comment:
is this 326 represented in a variable anywhere? These magic numbers seem fragile in the long run.

Reply:
I copied it from the sites dashboard CSS and it worked for me. I agree that it could be a variable somewhere.